packages/js_of_ocaml/js_of_ocaml.2.8.4/opam

```diff
@@ -12,12 +12,12 @@ depends: [
   "cmdliner"
   "base-unix"
   "ocamlfind" {>= "1.5.1"}
-  "lwt" {>= "2.4.4"}
+  "lwt" {>= "2.4.4" & < "4.0.0"}
   "menhir"
   "cppo" {>= "1.1.0"}
   "camlp4"
   "base64" {>= "2.0.0"}
-  ("base-no-ppx" | "ppx_tools")
+  "base-no-ppx" | "ppx_tools"
```

Comment:
@AltGr is this correct ?

Reply:
... it is, although I agree it looks pretty suspicious.
This disjunction is just one member of the `depends:` list, which is implicitely a conjunction. Explicit `&` conjunctions would have bound tighter than `|` though.
So  `["a" "b" | "c" "d"]` → a & (b | c) & d
but `["a" & "b" | "c" & "d"]` → (a & b) | (c & d)